ui/main-sidebar/main-sidebar.reel/main-sidebar.js

```diff
@@ -6,6 +6,10 @@ var Component = require("montage/ui/component").Component;
  */
 exports.MainSidebar = Component.specialize({
 
+    confirmationPromise: {
```

Comment:
@pchaussalet I think that got replaced with `_confirmationDeferred`, can you confirm?